site/docs/components/card/usage.mdx

```diff
@@ -8,31 +8,34 @@ data:
   $ref: ./#/data
 ---
 
-### Using the components
+### Using the component
+
+Cards may include text, images, or interactive elements. Make sure the content aligns with your use case, keeping it clear and easy to read. Avoid overcrowding cards with excessive content, actions, or links embedded in the text.
 
 #### When to use
 
 - To group related information with the same visual hierarchy.
 - To present a summary of a larger idea.
 - To link to more detailed information.
 
-#### When not to use
+#### When not to use
```

Comment:
MDX not reading this as a space for some reason so styles not being applied - space just needs to be removed and re-added.
Same on lines 27 + 33

Reply:
ty